### ISSUE TITLE
Edit Definition "Log-normal distribution"

### ISSUE THREAD

**Issue by JoramSoch** [StatProofBook/StatProofBook.github.io#147]
I'd suggest to **not** define the [log-normal distribution](https://statproofbook.github.io/D/lognorm) via its PDF, but with recourse to the normal distribution. That is, something like this: "Let $\ln X$ follow a [normal distribution](https://statproofbook.github.io/D/norm) ... Then, $X$ is said to be log-normally distributed ...". Based on this, the PDF of the log-normal distribution could be proven [along these lines](https://en.wikipedia.org/wiki/Log-normal_distribution#Probability_density_function). This has been done in similar way for the [t-distribution](https://statproofbook.github.io/D/t) and the [F-distribution](https://statproofbook.github.io/D/f). Additionally, the CDF of the log-normal distribution could be derived to support the proof showing the [median of the log-normal distribution](https://statproofbook.github.io/P/lognorm-med) where it's being cited.

To summarize:
* edit: [Log-normal distribution](https://statproofbook.github.io/D/lognorm)
* write: [Probability density function of the log-normal distribution](https://statproofbook.github.io/P/lognorm-pdf)
* write: [Cumulative distribution function of the log-normal distribution](https://statproofbook.github.io/P/lognorm-cdf)

**Comment by JoramSoch** [comment]
@majapavlo

**Comment by majapavlo** [comment]
Will look into these

**Comment by JoramSoch** [comment]
Resolved with commits [d322774](https://github.com/StatProofBook/StatProofBook.github.io/commit/d322774405863a56a74c66bbe031ed37e4538586) and [429c9ca](https://github.com/StatProofBook/StatProofBook.github.io/commit/429c9ca2634ccbc6cdea29d76a8580409f11645b).
